pkg/armrpc/frontend/server/handler.go

```diff
@@ -128,6 +128,25 @@ func HandlerForController(controller ctrl.Controller, operationType v1.Operation
 	}
 }
 
+// CreateHandler creates an http.Handler for the given resource type and operation method.
```

Comment:
For reviewers, here's a suggested tour:

- Start with the `.tsp`, converters, and datamodel. That will give you an idea of the API surface being added.
- Next look at the frontend changes. This is where I implemented support for child-resources.
- Next look at the routing changes in UCP. It was impossible to implement the correct behavior using the current design of routing. UCP's requirements are more complex.
- Last take a look at the integration tests + backend conrollers.